python-rhsm.spec

```diff
@@ -53,6 +54,18 @@ A small library for communicating with the REST interface of a Red Hat Unified
 Entitlement Platform. This interface is used for the management of system
 entitlements, certificates, and access to content.
 
+
+%package certificates
+Summary: Certificates required to communicate with a Red Hat Unified Entitlement Platform
+Group: Development/Libraries
+Version: 1.0.0
```

Comment:
We don't want to version these independently.  It will be much easier just to keep this package in lock-step with python-rhsm.  I don't think Tito even has the capability to maintain different version numbers in one spec file.

Just delete the version line entirely and it will inherit from the top.